app/OpeningCinematicPage/page.js

```diff
@@ -9,14 +9,14 @@ export default function OpeningCinematicPage() {
         <Wrapper>
             <MovieDisplay/>
             <TextBox>
-              <NextTxtButton> &#11208; </NextTxtButton>
+              <NextTxtButton>  </NextTxtButton>
```

Comment:
Let's change the name of this component.

I see room for improvement in the following aspects:
1) We should refrain from omitting vocals in names of functions, variables and so on (I've read about this in quite a few blogs now).
2) The name itself could be more descriptive.

My suggestions:
on 1) make it at least `NextTextButton`, or consider changing the name alltogether.

on 2)
Improve name of the component by:
- including an action verb, like: `ContinueTextButton`
- using a name to reflect component structure: `TextBoxButton` (could be the most straight-forward as we most likely won't have any more buttons in the text box).

Reply:
Good Idea, I changed it ;)

Reply:
please review ;)